src/sentry/api/urls.py

```diff
@@ -412,6 +413,7 @@
         AcceptOrganizationInvite.as_view(),
         name="sentry-api-0-accept-organization-invite",
     ),
+    url(r"^request-join/$", RequestJoinOrganization.as_view(), name="sentry-api-0-request-join"),
```

Comment:
This probably belongs under `organizations/<orgname>/request-join/`

Reply:
Hm what's your reasoning? I don't think the other unauth'ed endpoints are under `/organizations/`.

Reply:
I don't think it matters too much whether an endpoint is authed or unauthed. This endpoint is specific to an organization, so I think that it makes sense to have it under that url space. This also works well with `OrganizationEndpoint`, since the slug is part of the path. Looks like you've already moved this under organizations?